docs/reference/run.md

```diff
@@ -500,7 +500,8 @@ Or, to get the last time the container was (re)started;
 
 You cannot set any restart policy in combination with
```

Comment:
@afeld Thanks for the contribution.  Using the actual verb combine rather than the prepositional phrase tightens the sentence and brings the action a bit closer. I think your structure needs to lead with the flag rather than what it means. 

```markdown
You cannot combine the `--restart` (restart policy) flag with the [`--rm` (clean up)](#clean-up-rm) flag. Setting both `--restart` and `--rm` results in an error.  On container restart, attached clients are disconnected.
```